app/views/mechanics/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<section id="mechanic_details">
+<h1>Mechanic Details</h1>
+  <p><strong>Mechanic: </strong><%= @mechanic.name %></p>
+  <p><strong>Years of Experience: </strong><%= @mechanic.years_experience %></p>
+</section>
+
+<section id="ride_queue">
+ <h2>Current rides they are working on:</h2>
+
+  <% @mechanic.rides.each do |ride| %>
+    <ul>
+      <li><%= ride.name %></li>
+    </ul>
+  <% end %>
+</section>
+
+<section id="add_to_queue"> 
+  <%= form_with model: RideMechanic, data: {turbo: false} do |form| %>
```

Comment:
ooh form_with model! Nice! If you use form_with :model, you'll want to add an empty @ride_mechanic object in your show controller action so that you can pass that to the model parameter rather than the class name (i.e. `form_with model: @ride_mechanic`

form_with url is also a great option if form_with model gives you any problems on an IC